src/devices/Ws2812B/BitmapImageNeo3.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.Bindings.Utils;
+
+namespace Iot.Device.Bindings.WS2812B
+{
+    /// <summary>
+    /// Special 24bit RGB format for Neo pixel LEDs where each bit is converted to 3 bits.
+    /// A one is converted to 110, a zero is converted to 100.
+    /// </summary>
+    public class BitmapImageNeo3 : BitmapImage
+    {
+        const int BytesPerComponent = 3;
+        const int BytesPerPixel = BytesPerComponent * 3;
+        const int ResetDelayInBytes = 30; // 100us @ 2.4Mbps
+
+        public BitmapImageNeo3(int width, int height)
+            : base(new byte[width * height * BytesPerPixel + ResetDelayInBytes], width, height, width * BytesPerPixel)
+        {
+        }
+
+        public void SetPixel(int x, int y, uint color) => SetPixel(x, y, new Color(color));
+        public void SetPixel(int x, int y, byte level) => SetPixel(x, y, new Color(level));
+
+        public override void SetPixel(int x, int y, Color c)
```

Comment:
this coincidentally exactly matches the signature I'm using in the LED matrix for SenseHat (except the Color where I use the System.Drawing.Color) - we should create an issue to figure out the common abstraction

Reply:
See above, add your requirements to issue #189.